From fb8ce23a5ebf3e2b5f6cf1db9bf756c0324e50c6 Mon Sep 17 00:00:00 2001
From: Jiaxun Yang <jiaxun.yang@flygoat.com>
Date: Fri, 18 Oct 2024 20:51:39 +0800
Subject: [PATCH 169/301] AOSCOS: MIPS: Temporarily disable Loongson3 LL/SC
 errata check

To workaround the following errors:

  LLSCCHK vmlinux
ffffffff848a10c0: Branch target not a sync
(lots of these errors)
loongson3-llsc-check returns failure

Fixes: e4acfbc18fc9 ("MIPS: Check Loongson3 LL/SC errata workaround correctness")
Signed-off-by: Jiaxun Yang <jiaxun.yang@flygoat.com>
Link: https://t.me/c/1109254909/582977
Signed-off-by: Kexy Biscuit <kexybiscuit@aosc.io>
---
 arch/mips/Makefile.postlink | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/arch/mips/Makefile.postlink b/arch/mips/Makefile.postlink
index 6cfdc149d3bc..22b151c343b0 100644
--- a/arch/mips/Makefile.postlink
+++ b/arch/mips/Makefile.postlink
@@ -24,9 +24,6 @@ quiet_cmd_relocs = RELOCS  $@
 
 vmlinux: FORCE
 	@true
-ifeq ($(CONFIG_CPU_LOONGSON3_WORKAROUNDS),y)
-	$(call if_changed,ls3_llsc)
-endif
 ifeq ($(CONFIG_RELOCATABLE),y)
 	$(call if_changed,relocs)
 endif
-- 
2.48.1

